projects/distribution/distribution/Makefile

```
@@ -0,0 +1,29 @@
+BASE_DIRECTORY=$(shell git rev-parse --show-toplevel)
+GIT_TAG?=$(shell cat GIT_TAG)
+GOLANG_VERSION?="1.17"
+
+REPO=distribution
+REPO_OWNER=distribution
+
+BINARY_TARGET_FILES=registry
+SOURCE_PATTERNS=./cmd/registry
+
+VERSION=$(shell git describe --match 'v[0-9]*' --dirty='.m' --always)
+REVISION=$(shell git rev-parse HEAD)$(shell if ! git diff --no-ext-diff --quiet --exit-code; then echo .m; fi)
+PKG=github.com/docker/distribution
+
+GO_LDFLAGS=-s -w -X $(PKG)/version.Version=$(VERSION) -X $(PKG)/version.Revision=$(REVISION) -X $(PKG)/version.Package=$(PKG)
+GOBUILD_TAGS=include_oss include_gcs
```

```suggestion
EXTRA_GOBUILD_FLAGS=-tags include_oss include_gcs
```

Quotes may be necessary around the two tags, which can be a bit of pain to make sure they flow through make correctly.